### ISSUE TITLE
 Animation toggle button does not provide a tooltip when a window is smaller or browser is magnified

### ISSUE THREAD

**Issue by maureenlholland** [mozilla/bedrock#15762]
## Description

[See bugzilla for details](https://bugzilla.mozilla.org/show_bug.cgi?id=1936870)\

[Changes requested](https://bugzilla.mozilla.org/show_bug.cgi?id=1936630#c2) from accessibility review

---

#### Success Criteria
- [ ] Providing a tooltip via HTML title attribute would be a sufficient indication of the label provided on-screen.

NOTE: there is also a [comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1936870#c1) that the pause button does not have the same hover behaviour as play button. This doesn't seem to be the case in my testing. We may need to ask for more information to recreate.
